### PR TITLE
[#233] Slides of Quality and MainMenu

### LINKED ISSUE #233: Presentation:  Quality and Main Menu

### PR DESCRIPTION
**Pull request by Albanovukelaj17**

### CODE REVIEW

File: presentation/presentation.md [hunk 1]
```diff
@@ -65,16 +66,138 @@ Z. Zukunft <code>Wer noch was braucht</code>
...
+<!-- _footer: "B-Team: Albano "-->
+<style scoped>
+  img.right {
+    float: right;
+    margin-left: 20px; 
+    margin-top: 0px;
+    max-width: 60%; 
+  }
+</style>
+<img src="assets/img/MainMenu/MainMenu_CreateLobby.png" alt="Main Menu Create Lobby" class="right" />
```
Comment (casqan) [inline presentation/presentation.md:91]: Marp provides native Markdown Layout Syntax: https://marpit.marp.app/image-syntax if this works properly then it doesn't need to be changed, but you might be able to achieve what you want easier with this

File: presentation/presentation.md [hunk 2]
```diff
@@ -65,16 +66,138 @@ Z. Zukunft <code>Wer noch was braucht</code>
...
+### Common Module
+
+- **Automatisierte Tests:** Getestet mit **JUnit** 
+- **Ansatz:**
+  - **Unit Tests:** Methoden lieferten korrekte Ergebnisse.
+  - **Edge Cases:** Null-Werte und Extreme getestet.
+
+---
+<!-- _footer: "B-Team: Albano "-->
+## 5.1 Wie wurde getestet (Teil 2)
```
Comment (casqan) [inline presentation/presentation.md:129]: Keep consistent spacing. i.e:

```md
---

<!-- ... -->

## 5.1 ....

```

Leave a new line between slides

**Review by casqan — DISMISSED** [review]

**Comment by casqan** [comment]
Nothing *needs* to be done, these things would just be neater if time permits

File: presentation/presentation.md (hunk 1 above)
Comment (Albanovukelaj17) [reply presentation/presentation.md:91]: I am gonna not that and try to change it tomorrow in a separate format and style issue. I will merge this so that I have the basic for our presentation tomorrow

File: presentation/presentation.md [hunk 3]
```diff
@@ -65,16 +66,138 @@ Z. Zukunft <code>Wer noch was braucht</code>
...
 
 ---
+<!-- _footer: "B-Team: Albano "-->
+# 5. Qualität
+
+ **Ziele:**
+  - Fehlerfreiheit und robuste Funktionen.
+  - Intuitive und reibungslose Spielerfahrung.
+  - Erweiterbarkeit des Codes für zukünftige Features.
+  - Kompatibilität auf allen Geräten.
```
Comment (WalkerLukas) [inline presentation/presentation.md:114]: Sounds liek we release on mobile and console

File: presentation/presentation.md [hunk 4]
```diff
@@ -65,16 +66,138 @@ Z. Zukunft <code>Wer noch was braucht</code>
...
+## 5.2 Was haben wir getestet (Teil 3)
+
+### Client Module
+
+- **UI Testing:**  
+  - HealthBar-Rendering und Updates  
+  - Charakterdarstellung und Stat-Overlays 
+  - Korrekte CommandPreview and TurnOutcome
+- **Scroll-Funktionalität:**  
+  Scrollbare UI-Komponenten wie Listen und Dialoge wurden geprüft.
```
Comment (WalkerLukas) [inline presentation/presentation.md:198]: Wouldnt write as much about scrolling
More important:
Rendering works as intented.
Game reacts to inputs as intended
Game loop works correctly.
Communication with server is correct and answers get processed correctly

**Review by WalkerLukas — CHANGES_REQUESTED** [review]
See comemnts

**Review by casqan — DISMISSED** [review]

**Review by WalkerLukas — APPROVED** [review]
LGTM
